docs/source/executing-operations/subscription-support.mdx

```diff
@@ -101,9 +101,11 @@ subscription:
         reviews: # Overrides settings for the 'reviews' subgraph
           path: /ws # Absolute path that overrides '/subscriptions' defined above
           protocol: graphql_ws # The WebSocket-based subprotocol to use for subscription communication (Default: graphql_ws)
+          heartbeat_interval: 10s # Optional, with 'enable' equal to '5s' (Default: disabled)
```

Comment:
This line is a bit confusing to me.   Could you help me understand it better so I can suggest an alternative?

Reply:
Ok, so it's disabled by default. 
If you specify human readable time it will be that time, e.g. `10s`
If you specify `enable` it will be `5s`.

Reply:
I rewrote entire comment, hope it's more clear now.